products/compute/terraform.yaml

```diff
@@ -233,6 +233,68 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         diff_suppress_func: 'compareSelfLinkOrResourceName'
       timeoutSec: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+  RegionBackendService: !ruby/object:Overrides::Terraform::ResourceOverride
```

Comment:
This is duplicated from the above `BackendService` as they share a nearly identical structure

Reply:
It's my experience that this is often untrue. While load balancing Regional / Global resources will share a common body, each will use only around 60-70% of the fields.